README.md

```diff
@@ -382,19 +398,16 @@ kubectl set image deploy/api api=tryretool/backend:X.Y.Z
 
 ### Heroku deployments
 
-To update a Heroku deployment that was created with the button above, you may first set up a `git` repo to push to Heroku
-
-```zsh
-heroku login
-git clone https://github.com/tryretool/retool-onpremise
-cd retool-onpremise
-heroku git:remote -a YOUR_HEROKU_APP_NAME
-```
-
-To update Retool (this will automatically fetch the latest version of Retool)
+To update a Heroku deployment that was created with the button above, you'll
+need to push the desired changes to the Heroku remote.
 
 ```zsh
-git commit --allow-empty -m 'Redeploying'
+# This assumes you already have the `heroku` CLI installed and are logged in
```

Comment:
Do you think it makes sense to have teams explicitly set their version number from the local copy of the repo, and then push those changes to Heroku remote? This seems safer and gives teams more control over their updates. 

ie. Navigate to or clone the repository, edit the version # in `./heroku/Dockerfile`, then push to heroku remote